gatsby-config.js

```diff
@@ -279,6 +281,61 @@ module.exports = {
         nodesPerFeedFile: Infinity,
       },
     },
+    {
+      resolve: 'gatsby-plugin-feed',
+      options: {
+        query: `
+          {
+            site {
+              siteMetadata {
+                siteUrl
+              }
+            }
+          }
+        `,
+        feeds: [
+          {
+            query: `
+              {
+                allMdx(filter: {fileAbsolutePath: {regex: "/docs/release-notes/"}}) {
+                  nodes {
+                    id
+                    slug
+                    html
+                    frontmatter {
+                      releaseDate
+                      subject
+                      version
+                    }
+                  }
+                }
+              }
+            `,
+            serialize: ({ query: { site, allMdx } }) =>
+              allMdx.nodes.map(({ frontmatter, slug, id, html }) => {
+                const date = parseISO(frontmatter.releaseDate);
+                const formattedDate = format(date, 'EE, dd LLL yyyy');
+                const time = '00:00:00 +0000'; // time necessary for RSS validity
+
+                return {
+                  guid: id,
```

Comment:
Does this `guid` need to be consistent across builds? I'm not sure if that `id` is stable or not. We might want to double check that just in case.

Reply:
I'm not actually 100% on what this guid is used for, but I noticed it in the feed on the old site.

If I recall correctly, I think that the Gatsby-generated-ID is based off the filepath. If so, this might be "stable" enough to use. It's also totally possible that nothing uses this ID, but I wanted to keep the feed format as similar to the existing setup as possible.

Reply:
@austin-schaefer (or @roadlittledawn) Do you happen to know what the `guid` is used for in the RSS feed?

Reply:
Ya for sure! If this value was important, I just wanted to make sure that we aren't going to screw something up if this value isn't stable.